src/lib/workspace/EDAWorkspace.scss

```
@@ -157,7 +157,12 @@
 
   .Entities {
     display: flex;
-    justify-content: center;
+    overflow-x: auto;
```

@asizemore About your comment on the `web-components` PR, you need to add the `max-width` to the outermost container that contains only the entity diagram, in this case the `Entities` class element (this kind of stuff is definitely hard to describe, lol). I also wouldn't do the full `100vw`, instead adding a bit of buffer to accommodate the margin on the left side of the page. I'd suggest:

```suggestion
    overflow-x: auto;
    max-width: 97vw;
```